src/test/data/JsonSerializableAddressBookTest/duplicateEateryAddressBook.json

```diff
@@ -2,11 +2,9 @@
   "eateries": [ {
     "name": "Alice Pauline",
     "address": "123, Jurong West Ave 6, #08-111",
-    "category": "Western",
     "tagged": [ "friends" ]
   }, {
     "name": "Alice Pauline",
-    "address": "4th street",
-    "category": "Western"
+    "address": "4th street"
```

Comment:
why u change this ah lmao soz

Reply:
oh tbh idk, but i dont think duplicate eatery is based on category anyway?